cypress/e2e/app_tests/forms/headerVerificationRegistration.cy.js

```diff
@@ -0,0 +1,18 @@
+import { verifyAgreeToTerms, verifyConfirmPasswordInputTitle, verifyEmailInputTitle, verifyHeaderTitle, verifyNameInputTitle, verifyPasswordInputTitle, verifyTermsAndConditionsText } from "../../../models/formsModel/headerVerificationRegistrationModel";
+
+describe('Verify header and titles', () => {
+    beforeEach(() => {
+        cy.visit('/auth/register');
```

Comment:
you can create file in models, smth like helpers.js or common.js and put function there, navigateToUrl with parameter url, and reuse this func